CIP-0054/README.md

```diff
@@ -0,0 +1,227 @@
+---
+CIP: 54
+Title: Cardano Smart NFTs
+Authors: Kieran Simkin
+Comments-URI: https://forum.cardano.org/t/cip-draft-cardano-smart-nfts/100470
+Status: Draft
+Type: Standards
+Created: 2022-05-18
+License: CC-BY-4.0
+---
+# Cardano “Smart NFTs”
+
+## Abstract
+
+This CIP specifies a standard for an API which should be provided to Javascript NFTs, it also defines some additions to the 721 metadata standard defined in [CIP-0025](https://github.com/cardano-foundation/CIPs/tree/master/CIP-0025) which allow an NFT to specify it would like to receive certain current information from the blockchain. 
```

Comment:
Anything below this paragraph seems to belong more the "Motivation" than the abstract.